File: buildingTime.py
f = open('data.txt', 'r')
target = open('time.txt','w')
data = f.readlines()
i=0
size = len(data)
second = []
minute = []
hour = []

for num in data:
    if i==0:
        second.append(0)
        minute.append(0)
        hour.append(0)
    else:
        print(i)
        print(int(num)//1000)
        
        second.append(int(num)//1000)
        if second[i] > 60:
            minute[i] = second[i]//60
            if minute[i] >60:
                hour[i] = minute[i]//60
                minute[i] = minute[i]%60
            second[i] = second[i]%60
    target.write(str(hour[i]))
    target.write("\t")
    target.write(str(hour[i]))
    target.write("\t")
    target.write(str(hour[i]))
    target.write("\t")
    i=i+1


    

File: buildingTime.m
load data.txt;

[m,n] = size(data);
second = zeros(m,n);
minute = zeros(m,n);
hour = zeros(m,n);
for i=1:m
    if i==1
        second(i,1)=0;
        minute(i,1)=0;
        hour(i,1)=0;
    else
        
        second(i,1)=idivide(data(i,1),1000);
        if second(i,1) > 60
            minute(i,1) = idivide(second(i,1),60);
            if minute(i,1) >60
                hour(i,1) = idivide(minute(i,1),60);
                minute(i,1) = minute(i,1);
            end
            second(i,1) = mod(second(i,1),60);
        end
    end
end